packages/protocol/migrations/23_elect_validators.ts

```diff
@@ -12,12 +12,39 @@ import { signMessage } from '@celo/utils/lib/signatureUtils'
 import { BigNumber } from 'bignumber.js'
 import { AccountsInstance, ElectionInstance, LockedGoldInstance, ValidatorsInstance } from 'types'
 
-const Web3 = require('web3')
+import Web3 = require('web3')
+import { TransactionObject } from 'web3/eth/types'
 
 function serializeKeystore(keystore: any) {
   return Buffer.from(JSON.stringify(keystore)).toString('base64')
 }
 
+let isGanache = false
+
+const extraKeys = [
```

Comment:
Please include a comment describing what these are. Do they need to match the keys ganache has? Is there any way for us to get this via RPC, or the truffle config instead?